scripts/eq_validator_check.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import sys
+
+import requests
+from requests.exceptions import RequestException
+
+logger = logging.getLogger(__name__)
+
+try:
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-questionnaire-validator/releases",
+        timeout=120,
+    )
+    if response.status_code == 200:
+        with open(".schema-validator-version", encoding="UTF-8") as f:
+            version = f.readlines()[0]
+        latest_tag = response.json()[0]["tag_name"]
+        if latest_tag != version:
+            logger.error("eq-questionnaire-validator is out of date.")
```

Comment:
I suggest showing the update instruction the same way eq-translations does. i.e `Update using ...`

Reply:
Edited logger error message with `Update using...` instructions.